### PR TITLE
support riakc_pb_socket:start(_link)/3

### PR DESCRIPTION
**Pull request by chvanikoff**
#22 implementation

### CODE REVIEW

File: lib/riak/connection.ex [hunk 1]
```diff
@@ -2,14 +2,14 @@ defmodule Riak.Connection do
   @doc """
   Create a linked process to talk with the riak server on host:port.
   """
-  def start_link(host \\ '127.0.0.1', port \\ 8087) do
+  def start_link(host \\ '127.0.0.1', port \\ 8087, args \\ []) do
```
Comment (edgurgel) [inline lib/riak/connection.ex]: `args` are not being used here.

**Comment by chvanikoff** [comment]
@edgurgel oops, I've squashed fix into the commit

**Comment by edgurgel** [comment]
LGTM

**Comment by drewkerrigan** [comment]
:+1:

**Pull request merged**